### PR TITLE
Feature/aws deploy

### PR DESCRIPTION
**Pull request by Awarua-**
Deployment for Concierge demo to AWS

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ os:
     - osx
 node_js:
     - "4"
+    - "6"
```
Comment (mrkno) [inline .travis.yml]: Why are we now testing on node 6? Aside from regressions in node, there will be nothing we don't pick up with 4 and 7

File: .travis.yml [hunk 2]
```diff
@@ -15,3 +16,72 @@ addons:
...
+      secret_access_key: $AWS_SECRET_KEY
+      bucket: concierge-demo
+      key: Concierge.zip
+      bundle_type: zip
+      application: ConciergeDemo-App
+      deployment_group: ConciergeDemo-DepGrp
+      region: us-west-2
+      on:
+        repo: concierge/Concierge
+        branch: feature/aws-deploy
```
Comment (mrkno) [inline .travis.yml]: need to change the branch

File: .travis.yml [hunk 3]
```diff
@@ -15,3 +16,72 @@ addons:
...
+      secret_access_key: $AWS_SECRET_KEY
+      local_dir: dpl_cd_upload
+      bucket: concierge-demo
+      region: us-west-2
+      skip_cleanup: true
+      acl: public_read
+      detect_encoding: true
+      on:
+        repo: concierge/Concierge
+        branch: feature/aws-deploy
```
Comment (mrkno) [inline .travis.yml]: need to change the branch

File: package.json [hunk 4]
```diff
@@ -1,47 +1,47 @@
...
-    "url": "git+https://github.com/concierge/Concierge.git"
-  },
-  "author": "Matthew Knox",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/concierge/Concierge/issues"
-  },
-  "homepage": "https://github.com/Concierge/Concierge"
-}
+    "name": "concierge-bot",
```
Comment (mrkno) [inline package.json]: I do like 4 spaces, but any reason why we are fixing an unrelated issue in this PR?

Comment (Awarua-) [inline package.json]: Blame my code editor

File: .travis.yml (hunk 2 above)
Comment (Awarua-) [inline .travis.yml]: It will run against master and feature/aws-deploy

File: .travis.yml (hunk 3 above)
Comment (Awarua-) [inline .travis.yml]: It will run against master and feature/aws-deploy

File: .travis.yml (hunk 1 above)
Comment (Awarua-) [inline .travis.yml]: fair point

File: .travis.yml (hunk 3 above)
Comment (mrkno) [inline .travis.yml]: however after merge, feature/aws-deploy will be deleted...

File: .travis.yml (hunk 2 above)
Comment (mrkno) [inline .travis.yml]: however after merge, feature/aws-deploy will be deleted...

File: .travis.yml (hunk 3 above)
Comment (Awarua-) [inline .travis.yml]: true

**Pull request merged**